packages/pwa/app/components/_app-config/index.jsx

```diff
@@ -87,9 +73,22 @@ const AppConfig = ({children, locals = {}}) => {
 }
 
 AppConfig.restore = (locals = {}) => {
-    // Parse the locale from the page url.
-    const locale = getLocale(locals) || DEFAULT_LOCALE
-    const currency = getPreferredCurrency(locale) || DEFAULT_CURRENCY
+    const path =
+        typeof window === 'undefined'
+            ? locals.originalUrl
+            : `${window.location.pathname}${window.location.search}`
+    const url = pathToUrl(path)
+    const site = resolveSiteFromUrl(url)
+
+    if (site) {
```

Comment:
In what case would we not have a valid site id?

Reply:
For multi-site to work properly, a site Id would not suffice. I'd expect the `resolveSiteFromUrl` to return a site or an error is thrown. We don't need the conditional check anymore, but we have the siteId inside the commerceApiConfig. 

IMO, there are two ways to handle that:
1. Do not include siteId in the config in the first place, when a project is generator, that value will become the defaultSite on the config. This will involve some changes for the generator
2. Keep that siteId in the config, but use that value as a fallback for the default site when resolving a site when nothing works. =>I go with this one as it is less involved the change from the generator. You can see the changes  [here](https://github.com/SalesforceCommerceCloud/pwa-kit/pull/391/commits/ea61f816327a9a2767d9edd205b91f81114d6102#diff-b8450b99e168b84ffd9747d6a4f9a5341f548549bd88febe9e268e0bf6a9b657R62)

Reply:
You're probably thinking too deep about this. I'm simply saying that `resolveSiteFromUrl` does 2 things, returns a siteId or throws and exception. So if it returned without throwing, then you have a site, and you don't need a conditional.

Reply:
I got rid of the conditional, but we still have the siteId in the comerceApiConfig (generated from user's answer during the project generation process). My question is do we want to do anything with it though? 🤔

Reply:
We had a chat about the `siteId` in the apiConfig. Eventually, all the config will be moved into the config folder.